operator/pkg/controller/cassandradatacenter/cassandradatacenter_controller.go

```diff
@@ -120,6 +120,56 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	dcAnnotation := "cassandra.datastax.com/datacenter"
```

Comment:
This is used as a label (named DatacenterLabel from the cassdctypes) often, should probably use that one and explain why it's an annotation here instead of label.